src/config/sentry.ts

```diff
@@ -25,8 +26,9 @@ export const generateSentryConfig = (): SentryConfigType => {
   }
 
   return {
-    dsn: (!window.Cypress && process.env.SENTRY_DSN) || '',
+    dsn: (!window.Cypress && process.env.REACT_APP_SENTRY_DSN) || '',
     environment: SENTRY_ENVIRONMENT,
     tracesSampleRate: SENTRY_TRACE_SAMPLE_RATE,
+    release: process.env.REACT_APP_VERSION || 'no version',
```

Comment:
Should we use `undefined` rather than `no version` to model the absence of value?

Reply:
Yes `undefined` is a good idea, i will update it.

Reply:
@spaenleh I was actually thinking about using the JS `undefined` value, i.e. not as a string (see [stringly-typed](https://wiki.c2.com/?StringlyTyped))

Reply:
Mhmm but in this case, we don't need a default value, because `process.env.REACT_APP_VERSION` does not exist then it will be `undefined` by default no ?

Reply:
Yes, but do you want to use a string because you want to set a default value to the html meta tag?

Reply:
The version tag in the HTML is auto-replaced by CRA. If `REACT_APP_VERSION` is not set it will just be the empty string. This here is just to configure which release is associated to the sentry events that are sent. Does that make sense ?